css/style.css

```
@@ -1,3 +1,449 @@
+}
+
+.promo {
+    font: inherit;
+
+    color: #ffffff;   
+}
+
+.promo h3 {
+    font-size: 24px;
+    font-weight: 700; 
+}
+
+.promo-material a,
+.promo-tool a,
+.promo-mechanism a,
+.promo-discount a,
+.promo-delivery a {
+    background-color: rgba(255, 255, 255, 0.1);
+    color: #ffffff;
+}
+
+.promo-material a:hover,
+.promo-material a:focus,
+.promo-tool a:hover,
+.promo-tool a:focus,
+.promo-mechanism a:hover,
+.promo-mechanism a:focus,
+.promo-discount a:hover,
+.promo-discount a:focus,
+.promo-delivery a:hover,
+.promo-delivery a:focus {
+    background-color: rgba(255, 255, 255, 0.2);
+}
+
+.promo-material a:active,
+.promo-tool a:active,
+.promo-mechanism a:active,
+.promo-discount a:active,
+.promo-delivery a:active {
+    background-color: rgba(255, 255, 255, 0.3);
+}
+
+.promo-material {
+    background-color: #ffbf47;
+}
+
+.promo-tool {
+    background-color: #3bbce3;
+}
+
+.promo-mechanism {
+    background-color: #dc91d8;
+}
+
+.promo-discount {
+    background-color: #8ed78f;
+}
+
+.promo-delivery {
+    background-color: #ffc047;
+}
+
+.popular-title {
+    background-color: #f9f5f0;
+}
+
+.catalog-item {
+    font-family: "PT Sans", Arial, sans-serif;
+    border: 1px solid #a9a9a9;
+}
+
+.catalog-item h3 {
+    font-size: 18px;
+    color: #000000;
+    text-align: center;
+    line-height: 1.1;
+}
+
+.catalog-item-price {
+    font-size: 17px;
+    font-weight: 700;
+}
+
+.price-old {
+    color: #a9a9a9;
+    text-decoration: line-through;
+}
+.price-current {
+    color: #ffffff;
+
+    background-color: #ee3643;
+}
+
+.catalog-item:hover,
+.catalog-item:focus {
+    box-shadow: 0px 10px 25px 0px rgba(41, 52, 73, 0.5);
+}
+
+.catalog-item:hover .catalog-item-image,
+.catalog-item:focus .catalog-item-image {
+    display: none;
+}
+
+.catalog-item-btn {
+    font-family: "Cuprum", Arial, sans-serif;
+    font-size: 14px;
+
+
+    display: none;
+}
+
+.catalog-item:hover .catalog-item-btn,
+.catalog-item:focus .catalog-item-btn {
+    display: block;
+}
+
+.btn-cart {
+    color: #ffffff;
+    text-transform: uppercase;
+
+    background-color: #63a63e;
+    border-radius: 3px;
+    border-bottom: 3px solid #367315;
+}
+
+.btn-cart:hover,
+.btn-cart:focus {
+    background-color: #5fbb2d;
+}
+
+.btn-cart:active {
+    background-color: #518534;
+    border-bottom: 3px solid #518534;
+}
+
+.btn-bookmark {
+    color: #32425c;
+    text-transform: uppercase;
+
+    background-color: #ffffff;
+    border-radius: 3px;
+    border: 3px solid #63a63e;
+}
+
+.btn-bookmark:hover,
+.btn-bookmark:focus {
+    border: 3px solid #32425c;
+}
+
+.btn-bookmark:active {
+    color: rgba(50, 66, 92, 0.302);
+    border: 3px solid #c1c6ce;
+}
+
+.manufacturer-item {
+    text-align: center;
+
+    border: 1px solid #e5e5e5;
+}
+
+.manufacturer-item:hover,
+.manufacturer-item:focus,
+.manufacturer-item:active {
+    box-shadow: 0px 10px 25px 0px rgba(41, 52, 73, 0.5);
+}
+
+.manufacturer-item:active img {
+    filter: alpha(Opacity=3);
```

достаточно только одного `opacity`
```suggestion
    opacity: 0.3;
```